source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/styles/community.applications.css

```diff
@@ -0,0 +1,2392 @@
+  }
+  
+  input[type="button"][disabled] {
+    background: none !important;
+  }
+  
+  .searchSubmit {
+    /* Layout */
+    width: 2rem;
+    min-width: 2rem;
+    height: 3.4rem;
+    padding: 1.1rem 1rem 1rem 0;
+    position: relative;
+    
+    /* Typography */
+    font-family: 'FontAwesome';
+    font-size: 1.1rem;
+    
+    /* Visual */
+    background: var(--template-hover-background);
+    border: none;
+    
+    /* Interactive */
+    cursor: pointer;
+  }
+  
+  #searchButton:hover {
+    color: var(--unraid-66-color);
+  }
+  
+  /* Menu and Navigation */
+  .menuHeader {
+    font-size: 2rem;
+    margin: 1rem 0;
+  }
+  
+  .menuItems {
+    position: relative;
+    top: 2rem;
+    height: auto;
+    align-items: stretch;
+    min-width: 14rem;
+    max-width: 14rem;
+  }
+  
+  .selectedMenu {
+    color: var(--unraid-66-color);
+    font-weight: bold;
+  }
+  
+  .hoverMenu {
+    color: var(--unraid-66-color) !important;
+  }
+  
+  /* Sidebar Navigation */
+  .sidebar {
+    /* Layout */
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 1.6rem;
+    width: 100%;
+    height: 100vh;
+    margin-bottom: 10px;
+    /* Display over any banner warning */
+    z-index: 1000;
+    opacity: 0.9;
+    
+    /* Visual */
+    background-color: var(--sidebar-close-background);
+    display: none;
+  }
+  
+  .sidebarClose {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100vh;
+  }
+  
+  .sidenav {
+    /* Layout */
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 1.6rem;
+    margin-bottom: 10px;
+    padding-top: 60px;
+    
+    /* Visual */
+    background-color: var(--sidebar-background);
+    color: var(--sidebar-text);
+    
+    /* Interactive */
+    overflow-x: hidden;
+    overflow-y: scroll;
+    transition: 0.5s;
+  }
+  
+  .sidenavHide {
+    width: 0;
+  }
+  
+  .sidenavShow {
+    width: 90%;
+    max-width: 70rem;
+  }
+  
+  /* Utility Classes */
+
+  
+  .ca_multiselect {
+    float: right;
+    cursor: pointer;
+    padding-right: 1rem;
+  }
+
+  .noClick {
+    cursor: default !important;
+  }
+  
+  /* Animations */
+  @keyframes cycleRed {
+    0% { color: inherit; }
+    50% { color: #882626; }
+  }
+  
+  /* Responsive Design */
+  @media screen and (max-width: 800px) and (orientation: landscape),
+         screen and (max-height: 300px) {
+    .mobileHolderFix {
+      margin-bottom: 2rem !important;
+    }
+  }
+
+
+  /* Awesomplete Autocomplete Component */
+  .awesomplete {
+    /* Layout */
+    display: inline-block;
+    position: relative;
+    
+    /* Typography */
+    color: red;
+  }
+  
+  .awesomplete > input {
+    display: block;
+  }
+  
+  .awesomplete > ul {
+    /* Layout */
+    position: absolute;
+    left: 0;
+    z-index: 1;
+    min-width: 100%;
+    box-sizing: border-box;
+    margin: 0.2em 0 0;
+    padding: 0;
+    
+    /* Visual */
+    background: linear-gradient(to bottom right, white, hsla(0, 0%, 100%));
```

Comment:
Use an existing var here for the colors if possible https://github.com/unraid/webgui/blob/3cfa60e726b4a9df8ab581dcc1174f966211d114/emhttp/plugins/dynamix/styles/default-color-palette.css